Shift3/boilerplate-client-react
# Removed `HolyGrail` component

## Changes
1. Deleted `HolyGrail component`.
2. Removed border styles in `Layout` component.

## Purpose
Since a new `Layout` component was added to replace the `HolyGrail`, that component can now be removed, along with the border styles that were only for demo purposes.

## Testing
Make sure to run the Node or Django server.

1. Pull in the changes to your local copy of this branch and run `yarn start`.
2. Make sure there are no color borders showing.
